### DISCUSSION TITLE
Getting Builders Limit Error while deploying. Can't seem to figure out what went wrong

### DISCUSSION THREAD

**Discussion by Prottoy2938** [vercel/community#1640] [Help]
### Summary

I'm getting this error on my vercel CLI when I'm trying to deploy my app:

`Error: Builder returned invalid routes: should NOT have more than 100 properties`

What does this mean?  I can't find any  documentation of this

**Comment by amyegan** [comment]
Hi @Prottoy2938. Are you able to share the repo or minimal reproduction so we can get a better look at what is going wrong?

**Reply by Prottoy2938** [reply]
Of course. Heres the project code: https://github.com/Prottoy2938/vercel-deployment-issue-calcsurf

**Reply by Prottoy2938** [reply]
Any update?
